Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m

```diff
@@ -173,6 +173,27 @@ - (NSOperation *)getCollectionsPage:(NSUInteger)page completion:(BUYDataCollecti
 	}];
 }
 
+- (NSOperation *)getCollectionsByIds:(NSArray<NSString *> *)collectionIds completion:(BUYDataCollectionsBlock)block
+{
+	return [self getCollectionsByIds:collectionIds page:0 completion:block];
```

Comment:
This is probably unclear. Asking for page 0 doesn't get page 1, it gets all pages. @mtaher Can you confirm?